src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -109,13 +109,13 @@ private void buildGeneralParameters() {
         context.put("build_time", formattedTime);
 
         // build number is not mandatory
-        String buildNumber = configuration.getBuildNumber();
-        if (StringUtils.isNotBlank(buildNumber) && configuration.isRunWithJenkins()) {
-            if (NumberUtils.isCreatable(buildNumber)) {
-                context.put("build_previous_number", Integer.parseInt(buildNumber) - 1);
-            } else {
-                LOG.log(Level.INFO, String.format("Could not parse build number: %1$s.", configuration.getBuildNumber()));
-            }
+        // the build number could contain " ". calling strip will return 'null' to
+        // buildNumber
+        String buildNumber = StringUtils.stripToNull(configuration.getBuildNumber());
+        if (StringUtils.isNumeric(buildNumber) && configuration.isRunWithJenkins()) {
+            context.put("build_previous_number", Integer.parseInt(buildNumber) - 1);
+        } else {
+             LOG.log(Level.INFO, String.format("Could not parse build number: %1$s.", configuration.getBuildNumber()));
```

Comment:
what happened here with while spaces do github could not smoothly compare those lines

Reply:
Keeping the else statement should stay because I didn't test against Jenkins 1.x

Reply:
can you leave this comment in source code so nobody removes this piece some day?